lib/i18n/hygiene/checks/html_entity.rb

```diff
@@ -8,22 +8,12 @@ module Hygiene
     module Checks
       class HtmlEntity < Base
         def run
-          puts "Checking for phrases that contain entities but probably shouldn't..."
-
           wrapper = I18n::Hygiene::Wrapper.new(locales: all_locales)
 
           keys_with_entities = I18n::Hygiene::KeysWithEntities.new(i18nwrapper: wrapper)
 
           keys_with_entities.each do |key|
```

Comment:
Ah, in the next PR I have the messages become more complex, so they are split across multiple lines within that block, no longer possible to one line without going off the side of my screen.

eg
```ruby
keys_with_entities.each do |key|
  message = MessageBuilder.new
    .title("Unexpected HTML Entity")
    .key(key)
    .location("TODO")
    .create()

  yield Result.new(:failure, message: message)
end
```